### PR TITLE
Azabbasi/code q

### PR DESCRIPTION
**Pull request by azabbasi**

### CODE REVIEW

File: sdk/modelsrepository/Azure.Iot.ModelsRepository/src/ResolverClient.cs [hunk 1]
```diff
@@ -111,8 +112,8 @@ public ResolverClient(Uri repositoryUri, ResolverClientOptions options)
         /// <exception cref="ResolverException">Thrown when a resolution failure occurs.</exception>
         /// <param name="dtmis">A collection of well-formed DTDL model Ids.</param>
         /// <param name="cancellationToken">The cancellationToken.</param>
-        [System.Diagnostics.CodeAnalysis.SuppressMessage("Usage", "AZC0004:DO provide both asynchronous and synchronous variants for all service methods.", Justification = "<Pending>")]
-        [System.Diagnostics.CodeAnalysis.SuppressMessage("Usage", "AZC0015:Unexpected client method return type.", Justification = "<Pending>")]
+        [SuppressMessage("Usage", "AZC0004:DO provide both asynchronous and synchronous variants for all service methods.", Justification = "<Pending>")]
```
Comment (azabbasi) [inline sdk/modelsrepository/Azure.Iot.ModelsRepository/src/ResolverClient.cs:115]: These will be removed in a future PR

**Review by digimaun — APPROVED** [review]

File: sdk/modelsrepository/Azure.Iot.ModelsRepository/tests/ClientTests.cs [hunk 2]
```diff
@@ -80,9 +81,9 @@ public void EvaluateEventSourceKPIs()
             Type eventSourceType = typeof(ResolverEventSource);
```
Comment (azabbasi) [inline sdk/modelsrepository/Azure.Iot.ModelsRepository/tests/ClientTests.cs:81]: These events will be phased out in future PRs

File: sdk/modelsrepository/Azure.Iot.ModelsRepository/src/Fetchers/FetchResult.cs [hunk 3]
```diff
@@ -6,7 +6,11 @@ namespace Azure.Iot.ModelsRepository.Fetchers
     internal class FetchResult
     {
         public string Definition { get; set; }
+
         public string Path { get; set; }
-        public bool FromExpanded => Path.EndsWith("expanded.json", System.StringComparison.InvariantCultureIgnoreCase);
```
Comment (drwill-ms) [inline sdk/modelsrepository/Azure.Iot.ModelsRepository/src/Fetchers/FetchResult.cs:10]: Former string differs from new constant in that new constant is prefixed with a period. Is that intended?

Comment (azabbasi) [reply sdk/modelsrepository/Azure.Iot.ModelsRepository/src/Fetchers/FetchResult.cs:10]: yes, I tested this and it should be expected to have a period

**Review by drwill-ms — APPROVED** [review]

**Pull request merged**